.github/workflows/build.yml

```diff
@@ -0,0 +1,92 @@
+---
+
+name: Build wheels
+on: push
+jobs:
+  manylinux_wheels:
+    name: Python ${{ matrix.python_tag }} ${{ matrix.platform }} wheels
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform:
+          - manylinux1_x86_64
+          - manylinux2014_x86_64
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Docker
+        run: sudo apt-get install -y --no-install-recommends docker
+      - name: Pull Docker image
+        run: |
+          DOCKER_IMAGE="quay.io/pypa/${{ matrix.platform }}"
+          echo "::set-env name=DOCKER_IMAGE::$DOCKER_IMAGE"
```

Comment:
FTR this syntax is deprecated (CVE) and doesn't work anymore

Reply:
I think it's fully disabled now, actually. The new syntax is cleaner, though. https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/

Reply:
Yep. But it doesn't matter now. It's already addressed in Matt's fork, he just didn't push it here.

Reply:
I think it's fully disabled now, actually. The new syntax is cleaner, just do `echo "{name}={value}" >> $GITHUB_ENV`.

Reply:
Exactly.

Reply:
https://github.com/nitzmahone/pyyaml/actions/runs/400413155/workflow#L30